usage/launcher/src/main/java/brooklyn/launcher/BrooklynWebServer.java

```diff
@@ -338,7 +346,10 @@ public synchronized void start() throws Exception {
         if (actualPort == -1){
             PortRange portRange = requestedPort;
             if (portRange==null) {
-                portRange = getHttpsEnabled()? httpsPort : httpPort;
+                portRange = managementContext.getConfig().getConfig(BrooklynWebConfig.WEB_CONSOLE_PORT);
+            }
+            if (portRange==null) {
+                portRange = getHttpsEnabled() ? httpsPort : httpPort;
```

Comment:
Backwards compatibility warning: if `brooklyn.webconsole.security.https.required=true` was used previously without explicit port, the server would still start on 8081.